src/slack_api_util.ts

```diff
@@ -71,6 +72,25 @@ export async function getThreadPermalink(
   }
 }
 
+export async function makeFilesPublic(files: SlackFile[]): Promise<void> {
+  for (const file of files) {
+    const response = await axios.post(
+      'https://slack.com/api/files.sharedPublicURL',
+      null,
+      {
+        params: {
+          file: file.id,
+          token: process.env.SLACK_USER_ACCESS_TOKEN,
+        },
+      }
+    );
+    if (!response.data.ok) {
+      logger.error(`SLACKAPIUTIL.makeFilesPublic failed with ${response.data}`);
```

Comment:
can you stringify `response.data`? I believe it otherwise appears as `[Object object]`